indexer/indexer_tests/locality_index_test.cpp

```diff
@@ -65,4 +65,78 @@ UNIT_TEST(LocalityIndexTest)
   TEST_EQUAL(GetIds(index, m2::RectD{0.5, -0.5, 1.5, 1.5}), (Ids{2, 3}), ());
   TEST_EQUAL(GetIds(index, m2::RectD{-0.5, -0.5, 1.5, 1.5}), (Ids{1, 2, 3, 4}), ());
 }
+
+using RankedIds = vector<uint64_t>;
+
+template <typename LocalityIndex>
+RankedIds GetRankedIds(LocalityIndex const & index, m2::PointD const & center,
+                       m2::PointD const & border, uint32_t topSize)
+{
+  RankedIds ids;
+  index.ForTopFromPoint([&ids](osm::Id const & id) { ids.push_back(id.EncodedId()); }, center,
+                        MercatorBounds::DistanceOnEarth(center, border), topSize);
+  return ids;
+};
```

Comment:
Let's move the using and the method above.

Reply:
done